modules/core/src/main/java/org/apache/ignite/internal/processors/datastructures/DataStructureType.java

```diff
@@ -58,13 +58,16 @@
     SEMAPHORE(IgniteSemaphore.class.getSimpleName()),
 
     /** */
-    REENTRANT_LOCK(IgniteLock.class.getSimpleName());
+    REENTRANT_LOCK(IgniteLock.class.getSimpleName()),
```

Comment:
may be u delete old reentrant lock, all corresponding types and so on.

Reply:
I can't.

Reply:
I could help u ^^

Reply:
I think this is a task for the other issue in Jira:-)